conf/variant/common/conf-notes.txt

```diff
@@ -0,0 +1,12 @@
+To specify which hardware to build for two manual steps are required:
+
+1. Set MACHINE in "conf/local.conf"
+2. Enable BSP and or other features such as QtAuto in "conf/bblayers.conf"
+
+Available PELUX images:
+* core-image-pelux-minimal
+* core-image-pelux-minimal-update
+
+* core-image-pelux-qtauto-neptune     (Requires QtAuto)
+
+Build them by running: bitbake <image>
```

Comment:
This will be printed after source:ing... but it is OK to modify conf/bblayers.conf after that and bitbake will do the correct thing? One can't change things in `conf/` that would require "re-source:ing"? I assume it is OK, just checking.

Reply:
sourcing just copies from `source/meta-pelux/conf/.../*.conf.sample` to `build/conf/*conf`
Meaning editing `build/conf/*conf` after sourcing is fine. No resourcing required

Even better this is the intended use and resourcing will __not__ have any effect (besides setting the build env and cd'ing to the build directory). To reset one needs to `rm build/conf/templateconf.cfg`

Reply:
OK, good.

Another thing just occurred to me though. How do we do this in a simple way in the CI?

`$ sed -i 's/# MACHINE=foo/MACHINE=foo/` conf/local.conf`

Reply:
Can the correct thing be done in `conf/bblayers.conf` depending on `MACHINE` perhaps? I do not know in which order things are parsed.

Reply:
> Can the correct thing be done in `conf/bblayers.conf` depending on `MACHINE` perhaps? I do not know in which order things are parsed.

No. AFAIK bblayers.conf does not share the same context as local.conf

Reply:
> OK, good.
> 
> Another thing just occurred to me though. How do we do this in a simple way in the CI?
> 
> ```shell
> $ sed -i 's/# MACHINE=foo/MACHINE=foo/` conf/local.conf
> $ sed -i 's/# require conf/sample/bar.conf/require conf/sample/bar.conf` conf/bblayers.conf
> ```
> 
> Ugh. :/

Uncommenting is already done by CI for ACCEPT_EULA when building for jetson-tx2

Reply:
And CI knows what machine is supposed to be used.

But it is most likely wise to think a bit about this before implementing

Reply:
Can you create a new Jenkins job which evaluates the new way of configuring and see how it works?

Reply:
I will make an attempt. Creating a WIP PR for pelux-manifests